onnxruntime/core/framework/bfc_arena.cc

```diff
@@ -230,11 +230,14 @@ void* BFCArena::Reserve(size_t size) {
     return nullptr;
 
   std::lock_guard<OrtMutex> lock(lock_);
+
+  LOGS_DEFAULT(INFO) << "Reserving memory in BFCArena for " << device_allocator_->Info().name << " size: " << size;
+
   void* ptr = device_allocator_->Alloc(size);
   ORT_ENFORCE(reserved_chunks_.find(ptr) == reserved_chunks_.end());
   reserved_chunks_.insert(std::pair<void*, size_t>(ptr, size));
   stats_.bytes_in_use += size;
-  stats_.num_allocs += 1;
+  stats_.num_reserves += 1;
```

Comment:
Mild change in behavior in that `num_allocs` will now not record number of calls to Reserve as there is another stat holding that info. This shouldn't affect anyone though.